### PR TITLE
Places: update UI elements on map

### PR DESCRIPTION
**Pull request by cooltey**
https://phabricator.wikimedia.org/T353562

### CODE REVIEW

File: app/src/main/res/drawable/baseline_circle_24.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="36dp" android:tint="#000000"
```
Comment (dbrant) [inline app/src/main/res/drawable/baseline_circle_24.xml:1]: If our marker is a simple circle, and we're already drawing other circles directly onto the `Canvas` of our markers, do we really need a separate vector graphic of a circle?

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/res/drawable/baseline_circle_24.xml (hunk 1 above)
Comment (cooltey) [reply app/src/main/res/drawable/baseline_circle_24.xml]: Sure! Done!

File: app/src/main/java/org/wikipedia/places/PlacesFragment.kt [hunk 2]
```diff
@@ -464,27 +480,31 @@ class PlacesFragment : Fragment(), MapboxMap.OnMapClickListener {
...
-            // Draw the marker frame on top of the clipped thumbnail.
-            drawBitmap(markerBitmapBase, markerBitmapBaseRect, markerRect, null)
+            drawBitmap(circularBitmapWithBorder(thumbnailBitmap), null, markerRect, null)
         }
         return result
     }
 
+    private fun circularBitmapWithBorder(thumbnailBitmap: Bitmap? = null): Bitmap {
+        val radius = MARKER_SIZE / 2f
+        val outputBitmap = Bitmap.createBitmap(MARKER_SIZE, MARKER_SIZE, Bitmap.Config.ARGB_8888)
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/places/PlacesFragment.kt:496]: This creates a `Bitmap` every time this function is called, and the `Bitmap` is never recycled. Bitmaps are special objects that must be [managed carefully](https://developer.android.com/topic/performance/graphics/manage-memory).

The only explicit bitmap instance should be `markerBitmapBase`.

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/places/PlacesFragment.kt [hunk 3]
```diff
@@ -92,10 +91,28 @@ class PlacesFragment : Fragment(), MapboxMap.OnMapClickListener {
     private var lastLocationUpdated: LatLng? = null
 
     private lateinit var markerBitmapBase: Bitmap
-    private lateinit var markerBitmapBaseRect: Rect
-    private val markerRect = Rect(0, 0, MARKER_WIDTH, MARKER_HEIGHT)
-    private val markerPaintSrc = Paint().apply { isAntiAlias = true; xfermode = PorterDuffXfermode(Mode.SRC) }
-    private val markerPaintSrcIn = Paint().apply { isAntiAlias = true; xfermode = PorterDuffXfermode(Mode.SRC_IN) }
+    private val markerRect = Rect(0, 0, MARKER_SIZE, MARKER_SIZE)
+    private val markerPaintSrc by lazy {
```
[the remark this answers is not in the document]
Comment (cooltey) [reply app/src/main/java/org/wikipedia/places/PlacesFragment.kt]: Done!

File: app/src/main/java/org/wikipedia/places/PlacesFragment.kt [hunk 4]
```diff
@@ -195,12 +211,12 @@ class PlacesFragment : Fragment(), MapboxMap.OnMapClickListener {
...
                 mapboxMap = map
 
-                style.addImage(MARKER_DRAWABLE, AppCompatResources.getDrawable(requireActivity(), R.drawable.map_marker)!!)
+                style.addImage(MARKER_DRAWABLE, markerBitmapBase)
 
                 // TODO: Currently the style seems to break when zooming beyond 16.0. See if we can fix this.
                 map.setMaxZoomPreference(15.999)
 
-                map.uiSettings.isLogoEnabled = false
+                map.uiSettings.isAttributionEnabled = false
```
[the remark this answers is not in the document]
Comment (cooltey) [reply app/src/main/java/org/wikipedia/places/PlacesFragment.kt]: Done!

File: app/src/main/java/org/wikipedia/places/PlacesFragment.kt (hunk 2 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/places/PlacesFragment.kt]: Not sure but can we use `Glide.get(requireContext()).bitmapPool.getDirty(MARKER_SIZE, MARKER_SIZE, Bitmap.Config.ARGB_8888)` just like what we did in the `getMarkerBitmap()`?

**Pull request merged**